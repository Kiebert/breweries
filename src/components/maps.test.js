import React from 'react';
import { shallow } from 'enzyme';
import Maps from './Maps';

describe('<Maps />', () => {
    it('renders correctly', () => {

        const addresses = [
            {
                "name": "Brouwerij Dampegheest",
                "address": "Achterweg 22",
                "zipcode": "1906 AG",
                "city": "Limmen",
                "open": [
                    "Friday",
                    "Saturday",
                    "Sunday"
                ]
            },
            {
                "name": "de naeckte brouwers",
                "address": "Weverij 5",
                "zipcode": "1185 ZE",
                "city": "Amstelveen",
                "open": [
                    "Wednesday"
                ]
            }
        ]
        const component = shallow(
            <Maps 
                addresses={addresses}
                zoom={7}
                ltn={52}
                lng={4} 
            />
        );
        expect(component).toMatchSnapshot();

    });

});