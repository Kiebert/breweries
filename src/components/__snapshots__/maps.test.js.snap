// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Maps /> renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="Map"
    id="map"
  />,
  "nodes": Array [
    <div
      className="Map"
      id="map"
    />,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Maps
        addresses={
          Array [
            Object {
              "address": "Achterweg 22",
              "city": "Limmen",
              "name": "Brouwerij Dampegheest",
              "open": Array [
                "Friday",
                "Saturday",
                "Sunday",
              ],
              "zipcode": "1906 AG",
            },
            Object {
              "address": "Weverij 5",
              "city": "Amstelveen",
              "name": "de naeckte brouwers",
              "open": Array [
                "Wednesday",
              ],
              "zipcode": "1185 ZE",
            },
          ]
        }
        lat={52.0552809}
        lng={4}
        ltn={52}
        zoom={7}
      />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Maps {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "markers": Array [],
        "props": Object {
          "addresses": Array [
            Object {
              "address": "Achterweg 22",
              "city": "Limmen",
              "name": "Brouwerij Dampegheest",
              "open": Array [
                "Friday",
                "Saturday",
                "Sunday",
              ],
              "zipcode": "1906 AG",
            },
            Object {
              "address": "Weverij 5",
              "city": "Amstelveen",
              "name": "de naeckte brouwers",
              "open": Array [
                "Wednesday",
              ],
              "zipcode": "1185 ZE",
            },
          ],
          "lat": 52.0552809,
          "lng": 4,
          "ltn": 52,
          "zoom": 7,
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="Map"
          id="map"
        />,
        "_debugID": 2,
        "_renderedOutput": <div
          className="Map"
          id="map"
        />,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Maps
    addresses={
      Array [
        Object {
          "address": "Achterweg 22",
          "city": "Limmen",
          "name": "Brouwerij Dampegheest",
          "open": Array [
            "Friday",
            "Saturday",
            "Sunday",
          ],
          "zipcode": "1906 AG",
        },
        Object {
          "address": "Weverij 5",
          "city": "Amstelveen",
          "name": "de naeckte brouwers",
          "open": Array [
            "Wednesday",
          ],
          "zipcode": "1185 ZE",
        },
      ]
    }
    lat={52.0552809}
    lng={4}
    ltn={52}
    zoom={7}
  />,
}
`;
